### ISSUE TITLE
AimBlox exploits are detectable

### ISSUE THREAD

**Issue by whentheis** [Straden/Scripts#1]
I've been using the stroncc script for a few days, and today as soon as I injected the script, my roblox client just randomly crashed and when I joined the game back, I'm banned. 

I thought I was finally banned, so I tried another account.

Made an account, joined aimblox competitive, injected the script, and my roblox client crashed again. Joined back, and banned.

Tried again with more accounts, the same thing happened.
So AimBlox basically got an anti cheat update that detects your script, just wanna let you know

**Comment by whentheis** [comment]
Seems to be fixed, though now there are less options that it used to, is this normal? Or are the old options removed now?

**Comment by Straden** [comment]
Yep, silent aim had been detected and still working on a fix (im the only dev for the hub). Also it's easier for you to dm me on discord Throit#0699 for future updates and join my server :D

**Comment by whentheis** [comment]
Can you send the link? I clicked on the pop up that shows when executing the script, and it did nothing
Can't add you either, invalid username/discriminator

**Comment by Straden** [comment]
https://discord.gg/MWtXtGT3pJ
